src/ethereum/info/ethereumInfo.ts

```diff
@@ -1206,6 +1210,12 @@ const networkInfo: EthereumNetworkInfo = {
     chainId: 1,
     name: 'Ethereum Mainnet'
   },
+  decoyAddressConfig: {
```

Comment:
So you don't have to change so many files, these could have been default values that are only changed by chains that want something different.